CHANGELOG.md

```diff
@@ -1,9 +1,6 @@
 # Change history for ui-data-import
 
-## **6.0.1** (in progress)
-
-### Features added:
-* Add accessibiity testing to automated tests in ui-data-import (UIDATIMP-1372)
```

Comment:
Entry for `1372` will get lost - you can add a 6.1.0 version in progress and keep it there